packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -315,6 +352,7 @@ const GeneratedDesignPicker: React.FC< GeneratedDesignPickerProps > = ( {
 				} );
 				return (
 					<div className="design-button-container" key={ `generated-design__${ design.slug }` }>
+						<TrackDesignView design={ design } />
```

Comment:
We could wrap this part into `GeneratedDesign` and then we're able to use `useTrackDesignView` that I mentioned above.